pkgs/development/interpreters/octave/default.nix

```diff
@@ -185,15 +196,25 @@ in mkDerivation rec {
     cp test/fntests.log $out/share/octave/${pname}-${version}-fntests.log || true
   '';
 
-  passthru = {
+  passthru = rec {
+    octave = self;
     sitePath = "share/octave/${version}/site";
+    sitePackages = "share/octave/octave_packages";
     blas = blas';
     lapack = lapack';
     qrupdate = qrupdate';
     arpack = arpack';
     suitesparse = suitesparse';
     inherit python;
     inherit enableQt enableJIT enableReadline enableJava;
+    buildEnv = callPackage ./wrapper.nix {
+      octave = self;
+      inherit octavePackages;
+    };
+    withPackages = import ./with-packages.nix { inherit buildEnv octavePackages; };
+    pkgs = octavePackages;
+    interpreter = "${self}/bin/octave";
+    octPkgsPath = "share/octave/octave_packages";
```

Comment:
Actually, it was an accident. I was copying Python and Lua's respective definition files for their interpreters' `passthru` attribute set, and left that in. But, because octave doesn't use a `site-packages` directory for holding these packages, I defined a second variable `octPkgsPath` that had the exact same path.